tests/tests_public_logic.py

```diff
@@ -79,3 +79,29 @@ def test_multiple_track_different_arch_channel_map(self):
         }
 
         self.assertEqual(result, expected_result)
+
+    def test_build_os_info(self):
```

Comment:
Can you add just a comment before this function: 
```python
    # build_os_info
    # ===
```

Reply:
Done